src/commands/project/metadata.ts

```diff
@@ -0,0 +1,28 @@
+import { Arg } from "@oclif/core/lib/interfaces";
+import { TransactionCommand } from "../../base";
+import { getContract, getSigner, parseHash, pretty, run } from "../../helpers";
+
+export default class ProjectMetadata extends TransactionCommand {
+  static summary = "Set metadata property on a project.";
+  static examples = ['<%= config.bin %> <%= command.id %> \'{"property": "value"}\''];
+  static usage = "<%= command.id %> ID METADATA";
+  static args: Arg[] = [
+    { name: "ID", description: "The ID of the project to change metadata.", required: true },
+    {
+      name: "METADATA",
+      description: "New JSON metadata to attach to the project. Previous metadata will be overwritten.",
```

Comment:
Perhaps make it shorter so that it more readily fits on one line:

"New JSON metadata. Previous metadata will be overwritten."